www/js/alert-dialog.js

```diff
@@ -0,0 +1,97 @@
+
+
+define([
+	"dojo/_base/array",
+	"dojo/_base/declare",
+	"dojo/_base/lang",
+	"dojo/on",
+	"dojo/dom",
+	"dojo/dom-construct",
+	"dojo/query",
+	"dijit/registry",
+	"dojo/html",
+	"dojo/aspect",
+	"dijit/form/Button",
+], function(array, declare, lang, on, dom, domConstruct,  query, registry, html, aspect, Button) {
+	/* Summary:
+	 *				
+	 * 
+	 *			
+	 */
+	return declare(null, {
+		_clickHandlers: [],
+		_buttonsArray: [],
+
+		showDialog: function(title, alertContent, buttonsArray, cancelTitle){
+			alertDialog.set('title', title);
+			this._buttonsArray = buttonsArray;
+			d = dojo.byId('dialogButtons');
+			var cancelLabel = "Cancel";
+			if(cancelTitle && cancelTitle != ""){
+				cancelLabel = cancelTitle;
+			}
+
+			var closeButton = new Button({
+				label: cancelLabel,
+				id: 'cb'
+			});
+			var handler = on(closeButton, "click", lang.hitch(this, function(buttonsArray){
```

Comment:
lang.hitch() returns a function that will execute a given function in a given context. So, this works for multiple buttons.